clients/stellar-relay-lib/src/connection/helper.rs

```diff
@@ -6,19 +6,6 @@ use substrate_stellar_sdk::{
 	SecretKey, XdrCodec,
 };
 
-/// a helpful macro to log an error (if it occurs) and return immediately.
-macro_rules! log_error {
```

Comment:
Removed this because dropping the streams and connectors should be done before returning.